Pwin edit dcat 20190425

pwin opened the pull request:
editing and re-structuring; addition of string content to <a> tags where missing; removal of commented-out HTML

Still more to be done, but I don't want it to get too far behind other changes

awaiting outcome of #911 before changing text

pwin committed:
editing up to line 1111

pwin committed:
removal of chunks of the doc that are edited out

pwin committed:
editing

* removal of the commented-out HTML
* filling the <a> tags with string content where missing
* much editing and some minor re-structuring for clarity
Note: needs harmonising with the outcome of issue #911

dr-shorthair commented:
@pwin - I think you may have done a bunch of unnecessary work (and undone some earlier work by @andrea-perego ). 
Where a link's `href` points to a section anchor, the section number and title gets inserted automatically as the link text.

pwin committed:
Update index.html

reverted changes to the hyperlinks to internal anchors to fit with code @andrea-perego  added [whoops!  sorry @andrea-perego  :/ ]

pwin commented:
thanks for the heads-up @dr-shorthair 
I have reverted all cases where the hyperlink points to an internal anchor

dr-shorthair commented on dcat/index.html:
```diff
@@ -225,31 +224,23 @@ <h3>DCAT scope</h3>
         <li>
           <a href="#Class:Dataset"><code>dcat:Dataset</code></a> represents a dataset in a catalog.
           A dataset is a collection of data, published or curated by a single agent.
-          Data comes in many forms including numbers, words, pixels, imagery, sound and other multi-media, and potentially other types, any of which might be collected into a dataset.
```
Why remove this?

pwin replied:
@dr-shorthair I re-worked the preceding section about scope to include this point (see line 216) so that the definition of dcat:Dataset is cleaner

dr-shorthair reviewed (commented):
See one small question re. deletion at line 234

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -45,32 +45,32 @@
 
     <p>
-        The detailed differences between the two documents can be seen <a href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20180508%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20181011%2F">here</a>.
+        The detailed differences between Version 2 and Version 1 can be seen <a href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20180508%2F&doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FWD-vocab-dcat-2-20181011%2F">here</a>.
```
If we want to show the difference between dcat 2 and 1 then the link should be updated, we can't put the very last dcat2 dated URI as we haven't yet voted for the CR, but in the meanwhile,  we can use  "https://w3c.github.io/dxwg/dcat/" which is a good approximation, what do you think?
```suggestion
        The detailed differences between Version 2 and Version 1 can be seen <a href="https://services.w3.org/htmldiff?doc1=https://www.w3.org/TR/vocab-dcat/&doc2=https://w3c.github.io/dxwg/dcat/">here</a>.
```

andrea-perego replied:
Or maybe we can revise the text as follows:

> The detailed differences between [[VOCAB-DCAT-20140116]] and and this version of DCAT

andrea-perego commented on dcat/index.html:
```diff
@@ -45,32 +45,32 @@
     <h3 id="dcat_history">DCAT history</h3>
     <p>The original DCAT vocabulary was developed and <a href="http://vocab.deri.ie/dcat">hosted</a> at the Digital Enterprise Research Institute (DERI), then refined by the <a href="http://www.w3.org/egov/">eGov Interest Group</a>, and finally standardized in 2014 [[?VOCAB-DCAT-20140116]] by the <a href="http://www.w3.org/2011/gld/">Government Linked Data (GLD)</a> Working Group.</p>
-    <p>This revised version of DCAT was developed by the <a href="https://www.w3.org/2017/dxwg/">Dataset Exchange Working Group</a> in response to a new set of Use Cases and Requirements [[?DCAT-UCR]] gathered from peoples' experience with the DCAT vocabulary from the time of the original version, and new applications that were not considered in the first version. A summary of the changes from [[ ...
+    <p>This revised version of DCAT, Version 2, was developed by the <a href="https://www.w3.org/2017/dxwg/">Dataset Exchange Working Group</a> in response to a new set of Use Cases and Requirements [[?DCAT-UCR]] gathered from peoples' experiences with the DCAT vocabulary from the time of the original version, and new applications that were not considered in the first version. A summary of the ch ...
```
@pwin , I think the use of "Version 2" as in this line and others, is a bit redundant and maybe confusing. I would rather drop it, and just use "this version", or rather replace it with an abbreviation (e.g., "DCAT 2"), and include it in parenthesis. E.g., instead of:

> This revised version of DCAT, Version 2,

we could simply say

> This revised version of DCAT

or

> This revised version of DCAT (DCAT 2)

pwin replied:
the last option is the least ambiguous.  happy with that

andrea-perego reviewed (changes requested):
@pwin , about the use of "[in] the <a href="#section-id"></a> [section]", I think "the" and "section" are redundant and do not help reading. ReSpec already adds the § ("section") symbol, so no need to add "section". Moreover, ReSpec adds not only the section number, but also the section title (which sometimes is very long), so adding "section" at the end makes the text less fluent.

I would propose to roll back these changes.

andrea-perego reviewed (changes requested):
Another thing I would propose to fix:

The revisions use "Version 1" to refer to DCAT 2014. I would personally avoid this approach, and instead use the relevant bib ref ([[VOCAB-DCAT-20140116]] or simpy [[VOCAB-DCAT]]), which provides an unambiguous indication of the relevant DCAT version.

andrea-perego reviewed (commented):
@pwin , I saw you replaced cross-refs to examples with "as follows", "following". 

I think that it is preferable to provide an unambiguous link to an example, also when it follows the text. Anyway, if the other reviewers are happy with your approach, I won't object.

andrea-perego reviewed (changes requested):
I saw that some NOTEs stating that a property / class has been added etc. have been included at the end of the relevant section.

In a previous call, it was decided to move them just after the section title, and @riccardoAlbertoni started revising the spec accordingly, and so I've done in the subsequent PRs.

I suggest applying the same principle here.

andrea-perego commented:
> @pwin , I saw you replaced cross-refs to examples with "as follows", "following".
> 
> I think that it is preferable to provide an unambiguous link to an example, also when it follows the text. Anyway, if the other reviewers are happy with your approach, I won't object.

Sorry, @pwin , I now realise that you may not have done these changes, as the PR where I added the cross-refs to examples was created after yours. My apologies.

andrea-perego commented:
> I saw that some NOTEs stating that a property / class has been added etc. have been included at the end of the relevant section.
> 
> In a previous call, it was decided to move them just after the section title, and @riccardoAlbertoni started revising the spec accordingly, and so I've done in the subsequent PRs.
> 
> I suggest applying the same principle here.

Sorry, again. This issue is also probably due to PRs created after yours.

dr-shorthair commented:
This branch has not been synced with the master for a long time - is it still alive?

pwin commented:
It needs revisiting

On Tue, 23 Jul 2019, 06:01 Simon Cox, <notifications@github.com> wrote:

> This has drifted a long way from the master. Is it still alive?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/dxwg/pull/913?email_source=notifications&email_token=AAIFYTC3GVU6KMEB6OG7QI3QA2GDPA5CNFSM4HIW7UI2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2R5UVI#issuecomment-514054741>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIFYTDQTYQRIDHHRCQUOX3QA2GDPANCNFSM4HIW7UIQ>
> .
>

andrea-perego commented:
Moved this PR to the DCAT Ratification milestone, as it is editorial, and therefore can be sorted out after DCAT CR transition.

andrea-perego commented:
I suggest we close this PR, as the DCAT spec has changed substantively since then, so it is likely to need a newly created PR (time permitting).

pwin commented:
Agreed

On Mon, 28 Oct 2019, 00:12 Andrea Perego, <notifications@github.com> wrote:

> I suggest we close this PR, as the DCAT spec has changed substantively
> since then, so it is likely to need a newly created PR (time permitting).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/dxwg/pull/913?email_source=notifications&email_token=AAIFYTBHDUTZCAFND4SVAALQQYN7TA5CNFSM4HIW7UI2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOECLL45I#issuecomment-546750069>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIFYTCYPAIYUCIXPWBRBNTQQYN7TANCNFSM4HIW7UIQ>
> .
>

andrea-perego commented:
Thanks, @pwin .
